src/NuGet.Indexing/Querys/DownloadsScoreProvider.cs

```diff
@@ -81,7 +81,7 @@ private float AdjustByDownloads(int docId)
         public static float DownloadScore(long totalDownloads,
             QueryBoostingContext context)
         {
-            if (!context.BoostByDownloads)
+            if (context==null || !context.BoostByDownloads)
```

Comment:
Done.

---
In reply to: [75552402](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75552402) [](ancestors = 75552402)